paparazzi/paparazzi/src/test/java/app/cash/paparazzi/internal/resources/ValueResourceXmlParserTest.kt

```diff
@@ -3,63 +3,63 @@ package app.cash.paparazzi.internal.resources
 import com.android.ide.common.rendering.api.ResourceNamespace
 import com.google.common.truth.Truth.assertThat
 import org.junit.Test
-import org.xmlpull.v1.XmlPullParser
 import java.io.StringReader
 
 class ValueResourceXmlParserTest {
   @Test
   fun test() {
     StringReader(SIMPLE_LAYOUT).use {
       val parser = ValueResourceXmlParser().apply {
-        setFeature(XmlPullParser.FEATURE_PROCESS_NAMESPACES, true)
         setInput(it)
       }
 
+      parser.nextToken()
```

Comment:
necessary?  (why was this not needed before on the left?)

Reply:
As discussed , call `next()` will skip ENTITY_REF until reach next valid token, however, `nextToken()` will do one step, regardless of what the token is. But `next()` is disabled in `CommentTrackingXmlPullParser`, so we have to switch to `nextToken()`